client/src/node.rs

```diff
@@ -123,6 +136,49 @@ impl Node {
         Ok(())
     }
 
+    async fn get_payloads(&self, start_slot: u64, end_slot: u64) -> Result<Vec<ExecutionPayload>, NodeError> {
+        let payloads_fut = (start_slot..end_slot).rev().into_iter().map(|slot| async move {
```

Comment:
I made this concurrent to speed up the backfill, not sure if this might lead to rate limit failures on any public beacon api though and if it is worth adding some kind of "retry" logic for that case.

Reply:
I wouldn't worry about that for now. Most people use the default one I set, which doesn't rate limit. We can address the problem if it arises in the future.